opentreemap/stormwater/templates/stormwater/RainBarrel_add.html

```diff
@@ -0,0 +1,11 @@
+{% extends "treemap/resource_add.html" %}
+{% load i18n %}
+{% load l10n %}
+{% load form_extras %}
+
+{% block resource_add_details %}
+<div class="resource-question">
+  {% trans "How many gallons does the tank hold?" as capacity %}
```

Comment:
This label should avoid the term "gallons" so it will still make sense on a site using liters. Could change to "How much water does the tank hold?"